extra/pdf/layout/layout.factor

```diff
@@ -359,7 +359,7 @@ TUPLE: pdf-info title timestamp producer author creator ;
     pdf-info new
         now >>timestamp
         "Factor" >>producer
-        "USER" os-env "unknown" or >>author
+        "USER" os-env \ developer-name get-global "unknown" or or >>author
```

Comment:
Want to make that change? Or should I pull these commits and then make it myself?